src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/defense.json

```diff
@@ -7,10 +7,6 @@
     {
       "text": "Like many other modifier types, defense modifiers require defense slots to be applied. Typically, armor with a large number of defense slots start with fewer upgrade slots, meaning functionality is sacrificed for protection.",
       "paragraph": true
-    },
-    {
-      "text": "The Encyclopedia of Tinkering is the best source on defense modifiers, though Thruul plans to include them in later editions of Mighty Smelting.",
```

Comment:
I did not say delete this, I said update this. See the other modifier pages for examples, namely abilities